components/footer/src/index.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { spacing, typography } from '@govuk-react/lib';
+import { FOOTER_BACKGROUND, FOOTER_TEXT, FOOTER_BORDER_TOP } from 'govuk-colours';
+
+import WidthContainer from './atoms/width-container';
+import Navigation from './atoms/navigation';
+import Link from './atoms/link';
+import SectionBreak from './atoms/section-break';
+import MetaCustom from './atoms/meta-custom';
+
+import MetaContainer from './molecules/meta-container';
+import MetaLinks from './molecules/meta-links';
+import NavigationLinks from './molecules/navigation-links';
+
+const FooterContainer = styled('footer')(
+  {
+    borderTop: `1px solid ${FOOTER_BORDER_TOP}`,
+    background: `${FOOTER_BACKGROUND}`,
+    color: `${FOOTER_TEXT}`,
+  },
+  typography.font({ size: 16 }),
+  spacing.withWhiteSpace({ padding: { size: 7, direction: 'top' } }),
```

Comment:
Using two calls to `withWhiteSpace` isn't ideal - it would work, but override props would get interpreted/applied twice.

The value in `padding` can be an array - so this would be slightly better as:
```
spacing.withWhiteSpace({ padding: [{ size: 7, direction: 'top' }, { size: 5, direction: 'bottom' }] }),
```